generator/src/lib.rs

```diff
@@ -2805,20 +2817,25 @@ pub fn constant_name(name: &str) -> &str {
 }
 
 pub fn generate_constant<'a>(
-    constant: &'a vkxml::Constant,
-    cache: &mut HashSet<&'a str>,
+    constant: &'a vk_parse::Enum,
+    global_const_cache: &mut HashMap<&'a str, CType>,
 ) -> TokenStream {
-    cache.insert(constant.name.as_str());
-    let c = Constant::from_constant(constant);
+    let (c, _, is_alias) = Constant::from_vk_parse_enum(constant, None, None).unwrap();
+    // This requires knowing the type of the alias, that is unavailable here.  Besides
+    // backwards-compatibility, stabilization aliases serve no purpose.
```

Comment:
I'm not sure what this comment is trying to express -- it reads like a value judgement of the spec, but I assume it's actually intended to say something about the behavior of the following code?

It's especially confusing that it says "[the type of the alias] is unavailable" immediately before, apparently, looking up the type of the alias.